libs/vm/src/tokeniser.cpp

```diff
@@ -1908,7 +1908,7 @@ static int yy_get_next_buffer (yyscan_t yyscanner)
 			}
 		}
 
-	c = *(unsigned char *) yyg->yy_c_buf_p;	/* cast for 8-bit char's */
+	c = *(uint8_t *) yyg->yy_c_buf_p;	/* cast for 8-bit char's */
```

Comment:
In general we try not to mess with the VM tokeniser, as it's a generated file, effectively treated as third-party code.

Reply:
Unfortunately "Search and replace" did not know that. I will revert :)

Reply:
I am actually surprised that it does not use `flex_uint8`